ng-ncc-app/src/error.html

```diff
@@ -0,0 +1,64 @@
lip-path:none;clip-path:none;white-space:inherit}@media print{.govuk-skip-link{font-family:sans-serif}}.govuk-skip-link:focus{outline:3px solid #ffbf47;outline-offset:0;background-color:#ffbf47}.govuk-skip-link:active,.govuk-skip-link:focus,.govuk-skip-link:hover,.govuk-skip-link:link,.govuk-skip-link:visited{color:#0b0c0c}@media print{.govuk-skip-link:active,.govuk-skip-link:focus,.govuk-skip-link:hover,.govuk-skip-link:link,.govuk-skip-link:visited{color:#000}}.govuk-skip-link:link:focus{color:#0b0c0c}@media print{.govuk-skip-link:link:focus{color:#000}}@media (min-width:40.0625em){.govuk-skip-link{font-size:16px;line-height:1.25}}@media print{.govuk-skip-link{font-size:14pt;line-height:1.2}}.page{display:flex;flex-direction:column;flex-wrap:nowrap;height:100%}.page__header{position:relative;border-bottom:4px solid #00b140;background:#252525;color:#fff;padding:2rem}.page__header:after,.page__header:before{content:"";display:block;position:absolute}.page__header:before{background:#00664f;height:8px}.page__header:after{background:#a4d65e;height:2px}.page__header:after,.page__header:before{left:0;bottom:0;width:100%}.page__header:before{-webkit-transform:translate3d(0,-8px,1px);transform:translate3d(0,-8px,1px)}.page__header:after{-webkit-transform:translate3d(0,12px,1px);transform:translate3d(0,12px,1px)}.page__header p{color:inherit}@media (min-width:520px){.page__header__inner{display:flex;align-items:center}}.page__header__column--logo{flex-basis:300px}.page__header__column--title{flex-grow:1}.page__header__column--id{display:flex;flex-wrap:nowrap;align-items:center;justify-content:space-between}.page__header__logo{max-height:40px}.page__footer{position:relative;border-top:4px solid #00b140;background-color:#252525;color:#eee;margin-top:0;padding:1.5rem 2rem 1rem;text-align:center}.page__footer:after,.page__footer:before{content:"";display:block;position:absolute}.page__footer:before{background:#00664f;height:8px}.page__footer:after{background:#a4d65e;height:2px}.page__footer:after,.page__footer:before{left:0;top:0;width:100%}.page__footer:before{-webkit-transform:translate3d(0,8px,1px);transform:translate3d(0,8px,1px)}.page__footer:after{-webkit-transform:translate3d(0,-12px,1px);transform:translate3d(0,-12px,1px)}.app-container{display:flex;flex-grow:1;position:relative;padding-bottom:10px;min-height:0;min-width:0}.app-root{display:flex;flex-direction:column;height:100%}.app-content-area{flex-basis:60%;flex-grow:1;overflow:auto;padding:2rem;position:relative}.app-error-panel{background:0 0;border:4px solid #b10e1e;display:block;font-family:HelveticaNeue-Light,"Helvetica Neue Light","Helvetica Neue",Helvetica,Arial,"Lucida Grande",sans-serif;margin:1rem 0;padding:1rem;word-break:break-word}.app-error-panel .govuk-heading-m{margin-bottom:0}.govuk-\!-font-size-24 { font-size: 18px !important; line-height: 1.11111111 !important; }@media (min-width: 40.0625em) { .govuk-\!-font-size-24 { font-size: 24px !important; line-height: 1.25 !important; }}
```

Comment:
Why we are inlining this instead of using the CSS file we use for the rest of the project?